lib/Mojo/Base.pm

```diff
@@ -225,9 +225,13 @@ pass it either a hash or a hash reference with attribute values.
   $object = $object->tap($method);
   $object = $object->tap($method, @args);
 
-K combinator, tap into a method chain to perform operations on an object within
-the chain. The object will be the first argument passed to the callback, and is
-also available as C<$_>.
+Tap into a method chain to perform operations on an object within the
+chain (also known as a K combinator). The object will be the first
+argument passed to the callback and is also available as C<$_>. The
+callback's return value will be ignored; instead, the object (the
+callback's first argument) will be the return value. In this way,
+arbitrary code can be used within (i.e., spliced or tapped into) a
+chained set of object method calls, which is a commonly used pattern.
```

Comment:
This has come up on IRC, the "...which is a commonly used pattern" at the end, does not add much value to an otherwise good description.